crates/sb_graph/lib.rs

```diff
@@ -24,6 +26,8 @@ pub mod import_map;
 
 pub const VFS_ESZIP_KEY: &str = "---SUPABASE-VFS-DATA-ESZIP---";
 pub const SOURCE_CODE_ESZIP_KEY: &str = "---SUPABASE-SOURCE-CODE-ESZIP---";
+pub const STATIC_FILES_ESZIP_KEY: &str = "---SUPABASE-STATIC-FILES-ESZIP---";
+pub const STATIC_FS_PREFIX: &str = "mnt/data";
```

Comment:
should it be `/mnt/data`?

Reply:
It gets normalized later on, with `normalize_path`